spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -49,6 +49,11 @@ def run
 
       yaml = cookies.to_yaml.gsub("\n", "\\n")
 
+      if FastlaneCore::Helper.mac? && UI.interactive? && UI.confirm("🙄  Wow, that's a lot of text... should fastlane put it into your clipboard, so you can easily paste it?") 
+        copy_to_clipboard(yaml) 
```

Comment:
This method is defined in `environment_printer`, does this work even?

Reply:
Oh yah, so `spaceship` probably shouldn't just access a `fastlane` class, it's probably fair to copy that method over here, as it basically just shells out?

Reply:
> Find out if this is really the problem, if it is find another way to achieve what you want.

@janpio `UI` is not accessible here. What is the right way to use it?

Reply:
You are right. I have addressed it.

Reply:
I just tried and it seems to be working. Am I missing anything?